db/migrate/20190123130000_recipe_has_many_tags.rb

```diff
@@ -0,0 +1,8 @@
+# Create Join Recipe Tags Table
+
+class Recipe < ActiveRecord::Migration[5.2]
+  def change
+     has_many :tags
```

Comment:
Hmm. I think this might work, but it might be better to use the `has_and_belongs_to_many` association instead, as described in https://guides.rubyonrails.org/association_basics.html#the-has-and-belongs-to-many-association and https://guides.rubyonrails.org/association_basics.html#creating-join-tables-for-has-and-belongs-to-many-associations. In particular, I like the separate indexes on the join table, since you'll rarely be searching for them together.